.github/ISSUE_TEMPLATE/bug-report-advanced

```diff
@@ -0,0 +1,74 @@
+name: "\U0001F41E Bug report"
```

Comment:
Maybe add a mention somewhere that bug reports that do not follow the template and mandatory fields will be closed immediately. And if you want to explain why, a brief description along the lines of : this is to keep the developers focused on actually trobuleshooting & fixing bugs, then having to figure out with the user additional basic details everytime, as it currently happens, what blazorise version are you using, browser, code, etc...

Reply:
The form is already somewhat scary as it is to newcomers so don't want to scare them. Let's keep it like this for now.